src/rational.rs

```diff
@@ -113,6 +130,14 @@ impl Rational {
     }
 
     /**
+     * Returns this Rational to the nearest Int
+     */
+    pub fn round(&self) -> Int {
```

Comment:
Also, this function could probably check for a 0 `Rational` and return `Int::zero()` instead.